payments/card/seamless-view.md

```diff
@@ -384,6 +386,145 @@ embedded on your website.
+| `id`         | `string` | The relative URI to the payment.                                                                |
+| `instrument` | `string` | `Creditcard`                                                                                    |
+
+### `onPaymentCompleted`
+
+This event triggers when a payment has completed successfully.
+The `onPaymentCompleted` event is raised with the following event argument
+object:
+
+{:.code-header}
+**`onPaymentCompleted` event object**
+
+```js
+{
+    "id": "/psp/creditcard/payments/{{ page.paymentId }}",
+    "redirectUrl": "https://en.wikipedia.org/wiki/Success"
+}
+```
+
+{:.table .table-striped}
+| Property      | Type     | Description                                                     |
+| :------------ | :------- | :-------------------------------------------------------------- |
+| `id`          | `string` | The relative URI to the payment.                                |
+| `redirectUrl` | `string` | The URI the user will be redirect to after a completed payment. |
+
+### `onPaymentCanceled`
+
+This event triggers when the user cancels the payment.
+The `onPaymentCanceled` event is raised with the following event argument
+object:
+
+{:.code-header}
+**`onPaymentCanceled` event object**
+
+```js
+{
+    "id": "/psp/creditcard/payments/{{ page.paymentId }}",
+    "redirectUrl": "https://en.wikipedia.org/wiki/Canceled"
+}
+```
+
+{:.table .table-striped}
+| Property      | Type     | Description                                                    |
+| :------------ | :------- | :------------------------------------------------------------- |
+| `id`          | `string` | The relative URI to the payment.                               |
+| `redirectUrl` | `string` | The URI the user will be redirect to after a canceled payment. |
+
+### `onPaymentFailed`
+
+This event triggers when a payment has failed, disabling further attempts to
+perform a payment. The `onPaymentFailed` event is raised with the following
+event argument object:
+
+{:.code-header}
+**`onPaymentFailed` event object**
+
+```js
+{
+    "id": "/psp/creditcard/payments/{{ page.paymentId }}",
+    "redirectUrl": "https://en.wikipedia.org/wiki/Failed"
+}
+```
+
+{:.table .table-striped}
+| Property      | Type     | Description                                                  |
+| :------------ | :------- | :----------------------------------------------------------- |
+| `id`          | `string` | The relative URI to the payment.                             |
+| `redirectUrl` | `string` | The URI the user will be redirect to after a failed payment. |
+
+### `onPaymentTermsOfService`
+
+This event triggers when the user clicks on the "Display terms and conditions"
+link. The `onPaymentTermsOfService` event is raised with the following event
+argument object:
+
+{:.code-header}
+**`onPaymentTermsOfService` event object**
+
+```js
+{
+    "origin": "owner | merchant",
```

Comment:
No pipes in code examples, please. Describe the possible values in the property table below each example.

```suggestion
    "origin": "owner",
```